### DISCUSSION TITLE
Component contrast Issue

### DISCUSSION THREAD

**Discussion by ahmedabbous** [TDRA-ae/aegov-dls#17] [General]
The Alerts components in this link (https://designsystem.gov.ae/docs/components/alert) did not pass the contrast validation test. For example, the code below has insufficient contrast:

<div role="alert" class="aegov-alert alert-warning">
	<div class="alert-content">
		<p> Your password will expire in 3 days. Consider updating it now.</p>
	</div>
</div>

Should we use it?  Please advice!

**Comment by moiz-conceptlz** [comment]
A new update has been scheduled for the `aegov-dls` plugin. It will be released shortly which will include color changes to the alert components. 

If you have used the alert component as is in the structure indicated on the design system components documents, this update will require you to change nothing in your code. The class definitions will include new colours applied.
